composer.json

```
@@ -1,7 +1,7 @@
 {
     "require-dev": {
         "phpunit/phpunit": "9.6.12",
-        "automattic/vipwpcs": "2.3.4",
+        "automattic/vipwpcs": "3.0.0",
```

Now that [`6685e3e` (#4844)](https://github.com/Automattic/vip-go-mu-plugins/pull/4844/commits/6685e3ed2e9b765886ae3849b62bb91e8da56803) has been done, I think you'll want to update this to 3.0.1, since that's when the change to the error code happened.

```suggestion
        "automattic/vipwpcs": "3.0.1",
```

---

@GaryJones I think it is `wp-coding-standards/wpcs` that is updated, not `automattic/vipwpcs`.

https://github.com/Automattic/VIP-Coding-Standards/releases says that the latest version is 3.0.0

---

Oh, you're totally correct - apologies. 🤦🏻‍♂️